WinUIGallery/ControlPages/XamlFundamentals/XamlStylesPage.xaml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Page
+    x:Class="WinUIGallery.ControlPages.XamlStylesPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:WinUIGallery"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <StackPanel>
+        <StackPanel Spacing="12"
+                    Margin="0,12,0,0">
+            <RichTextBlock>
+                <Paragraph FontWeight="SemiBold">Placement of XAML Styles:</Paragraph>
+                <Paragraph>The placement of styles is the same as other resources: app-level, page-level, control-level.</Paragraph>
+            </RichTextBlock>
+
+            <RichTextBlock>
+                <Paragraph FontWeight="SemiBold">Key Points:</Paragraph>
```

Comment:
```suggestion

```

Do we need the header here? The above text is only a single line and this would fit with that?

Reply:
You are absolutely right